### DISCUSSION TITLE
Feature suggest: Compressor or limiter

### DISCUSSION THREAD

**Discussion by Gabry2790** [lavalink-devs/Lavalink#1057] [Q&A] [answered]
I think that a compressor or a limiter for audio is good, maybe for the equalizer that can make audio distorted (sorry for my english)

**Comment by duncte123** [comment, marked as answer]
you can use the volume filter to turn down the gain if it is too loud after EQ. A compressor will not fix that if the input of the filter is already clipping

**Reply by Gabry2790** [reply]
Ok, thanks
